projects/kit/components/routable-dialog/tui-wrap-in-lazy-dialog.ts

```diff
@@ -0,0 +1,19 @@
+import {Type} from '@angular/core';
+import {Route} from '@angular/router';
+import {TuiDialogOptions} from '@taiga-ui/core';
+
+import {TuiRoutableDialogComponent} from './routable-dialog.component';
+
+export function tuiWrapInLazyDialog<I>(
```

Comment:
I am not sure `wrapInDialog` is a good name

Reply:
Is `tuiGenerateDialogableRoute` ok?